src/librustc_mir/dataflow/move_paths/builder.rs

```diff
@@ -128,8 +130,10 @@ impl<'b, 'a, 'tcx> Gatherer<'b, 'a, 'tcx> {
                     ));
                 }
                 // move out of union - always move the entire union
+                // continue looking for other move errors though so that
```

Comment:
Was hard to read this comment wrt. where the sentence started and whatnot; would be good to add capitalization & punctuation.